src/superphot_plus/classify_ztf.py

```diff
@@ -381,7 +380,10 @@ class accuracy.
             ct += 1
 
             try:
-                tarr, farr, _, _ = import_data(os.path.join(data_dir, test_name + ".npz"))
+                lc = Lightcurve.from_file(os.path.join(data_dir, test_name + ".npz"))
```

Comment:
I also removed this remaining call to `import_data`, which has been removed in a Jeremy's recent [PR](https://github.com/lincc-frameworks/superphot-plus/pull/43)!